Missing dependency when upgrading to Polaris 5

## Issue summary

After upgrading to Polaris 5.0.0, it appears there's a missing dependency to `@shopify/browserslist-config`. This was tested with Create React App 3.4.1 using the `import '@shopify/polaris/dist/styles.css';` form of [getting Polaris styles](https://github.com/Shopify/polaris-react/blob/master/documentation/guides/migrating-from-v4-to-v5.md#build-output-changes---general-consumers) into your project.

## Expected behavior

After creating a new Create React App project and adding `@shopify/polaris` to your dependencies. Creating a basic Polaris app results in an error due to a missing dependency on `@shopify/browserslist-config`. It's expected that will work without that dependency.

## Actual behavior

The following error comes up when starting the application (`yarn start`):

```
./node_modules/@shopify/polaris/dist/styles.css (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./node_modules/@shopify/polaris/dist/styles.css)
Error: Cannot find module '@shopify/browserslist-config'
Require stack:
- /Users/testuser/Downloads/testing/node_modules/browserslist/node.js
- /Users/testuser/Downloads/testing/node_modules/browserslist/index.js
- /Users/testuser/Downloads/testing/node_modules/postcss-browser-comments/index.cjs.js
- /Users/testuser/Downloads/testing/node_modules/postcss-normalize/index.cjs.js
- /Users/testuser/Downloads/testing/node_modules/react-scripts/config/webpack.config.js
- /Users/testuser/Downloads/testing/node_modules/react-scripts/scripts/start.js
    at Array.reduce (<anonymous>)
```

If you perform `yarn add @shopify/browserslist-config -D`, the problem goes away. I'm not sure if this is expected and not documented (or I missed it) or if the dependency should be added as a devDependency/dependency in Polaris itself.

## Steps to reproduce the problem

1. yarn create react-app testing
1. cd testing
1. yarn add @shopify/polaris
1. Modify the index.js to match the Reduced test case below
1. yarn start
1. Above error is in the console and in the browser

## Reduced test case

```jsx
import '@shopify/polaris/dist/styles.css';
import React from 'react';
import ReactDOM from 'react-dom';
import enTranslations from '@shopify/polaris/locales/en.json';
import {AppProvider, Button} from '@shopify/polaris';

ReactDOM.render(
  <React.StrictMode>
    <AppProvider i18n={enTranslations}>
      <Button onClick={() => alert('Button clicked!')}>Example button</Button>
    </AppProvider>
  </React.StrictMode>,
  document.getElementById('root')
);

```

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 5.0.0
- Browser: Any
- Device: MacBook Pro
- Operating System: macOS High Sierra

--- a/UNRELEASED.md
+++ b/UNRELEASED.md
@@ -11,6 +11,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Bug fixes
 
 - Fixed case where `DatePicker` did not translate the weekday name in an aria label ([#3113](https://github.com/Shopify/polaris-react/pull/3113))
+- Updated browserslist config to be an explicit list instead of extending an existing config, so that consuming apps don't need to depend upon `@shopify/browserslist-config` ([#3132](https://github.com/Shopify/polaris-react/pull/3132))
 
 ### Documentation
 

--- a/package.json
+++ b/package.json
@@ -168,7 +168,13 @@
     "typescript": "~3.9.2"
   },
   "browserslist": [
-    "extends @shopify/browserslist-config"
+    "last 3 chrome versions",
+    "last 3 chromeandroid versions",
+    "last 3 firefox versions",
+    "last 3 opera versions",
+    "last 2 edge versions",
+    "safari >= 10",
+    "ios >= 10"
   ],
   "prettier": "@shopify/prettier-config",
   "stylelint": {

